.github/workflows/ci.yml

```diff
@@ -28,15 +28,13 @@ jobs:
           - ubuntu-latest
           - macOS-latest
           - windows-latest
-        arch:
-          - x64
```

Comment:
This is also a problem for ClimaOcean GitHub @navidcy

Reply:
What was the original point of specifying architecture ?

Reply:
Yeah, arch is there because the default `.yml` file included it.